src/React InstantSearch/src/App.css

```diff
@@ -10,24 +15,31 @@ em { background: cyan; font-style: normal }
 .twoColumns {
   display: grid;
   grid-template-columns: 20% 75%;
-  grid-gap: 5%
+  grid-gap: 5%;
 }
-.ais-Hits__root {
+
+.ais-Hits-list {
   display: grid;
-  grid-template-columns: 50% 50%;
+  grid-template-columns: 47.5% 47.5%;
```

Comment:
If those changes are also valid for other templates, can you do those changes on other templates too?

Reply:
Yes we can, it will avoid the children to be larger than the parent. I will do another PR.